### PR TITLE
sasview 5.0.6rc2 updates

### PR DESCRIPTION
**Pull request by wpotrzebowski**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml
Comment (mrakitin) [inline recipe/meta.yaml:51]:
```suggestion
    - {{ pin_compatible('numpy', max_pin="x.x") }}  # Maintainer note: numpy<1.24 seems to still be required
```

**Comment by mrakitin** [comment]
@wpotrzebowski, the fixed pin seemed to resolve the original issue, but now we are blocked by https://github.com/conda-forge/bumps-feedstock/pull/12. I'll need to confirm with other maintainers that it's OK with them to pin numpy<1.24.

By the way, did you observe the same issue with the sasview & numpy 1.24?
```py
AttributeError: module 'numpy' has no attribute 'MachAr'
```

**Comment by wpotrzebowski** [comment]
Thanks @mrakitin! 

The error I've encountered was different: `AttributeError: module 'numpy' has no attribute 'float'`

**Comment by wpotrzebowski** [comment]
Hi @mrakitin. Any update on numpy case?

File: recipe/meta.yaml
Comment (pbeaucage) [inline recipe/meta.yaml:55]: Should this have an actual max version instead of 'x.x'?  It does seem to work so maybe doesn't matter?

Comment (wpotrzebowski) [reply recipe/meta.yaml:55]: @pbeaucage Sorry, I missed it. My understanding is that values for x.x are taken from the "host" (https://github.com/conda-forge/sasview-feedstock/blob/d1e591cb22ae5b24de56e54c98efdf6030b64820/recipe/meta.yaml#L28)

**Comment by wpotrzebowski** [comment]
Any objections to merging it?

**Comment by pbeaucage** [comment]
Sorry, no objections from me, missed earlier message.

**Review by pbeaucage — APPROVED** [review]
LGTM!

**Pull request merged**
